### DISCUSSION TITLE
posts/22.-%ED%95%A8%EC%88%98-%EB%A7%8C%EB%93%A4%EA%B8%B0-1-(defun)-1/

### DISCUSSION THREAD

**Discussion by giscus** [IsaacGoCoder/IsaacGoCoder.github.io#5] [General]
# posts/22.-%ED%95%A8%EC%88%98-%EB%A7%8C%EB%93%A4%EA%B8%B0-1-(defun)-1/

목표 함수를 만드는 함수인 (defun)을 알아봅시다

https://isaacgocoder.github.io/posts/22.-%ED%95%A8%EC%88%98-%EB%A7%8C%EB%93%A4%EA%B8%B0-1-(defun)-1/

**Comment by yeonmech** [comment]
안녕하세요 올려주신 강의 글들 잘 읽었습니다
홈페이지 이설 하신 후에도 계속 올려 주시면 좋겠습니다
잘 배우고 갑니다, 감사합니다

**Reply by GoCoder7** [reply]
감사합니다!
